src/sidebar/components/menu-section.js

```diff
@@ -0,0 +1,51 @@
+'use strict';
+
+const { Fragment, createElement } = require('preact');
+const propTypes = require('prop-types');
+
+const map = val => (Array.isArray(val) ? val : [val]);
```

Comment:
This is a mistake I just spotted - it returns the input instead of mapping over the input array.

Reply:
When you write something like this:

```js
<MenuSection>
   <MenuItem>...</MenuItem>
</MenuSection>
```

Then `MenuSection` will be called with a `children` value which is a single object (specifically the value of `<MenuItem .../>`). If on the other hand there are multiple children, as in:

```js
<MenuSection>
   <MenuItem>...</MenuItem>
   <MenuItem>...</MenuItem>
</MenuSection>
```

Then `MenuSection` will be called with a `children` prop which is an array of two items. I believe avoiding an array in the single-child case is done as an optimization.